src/components/Dialog/Dialog.st.css

```diff
@@ -51,23 +72,17 @@
     transform: none;
 }
 
-.closeButtonWrapper {
-    position: absolute;
-    top: 20px;
-    right: 20px;
-}
-
 .root:rtl .closeButtonWrapper {
     right: initial;
     left: 20px;
 }
 
-.closeIconButton {
-    -st-mixin: TPAIconButton(
-            IconColor value(CurrentCloseButtonColor)
-    );
+.root:wired .contentWrapper {
+    background: value(WiredBackgroundColor);
 }
 
-.dialogContent {
-    padding: 80px 32px;
+.root:wired .closeIconButton {
+    -st-mixin: TPAIconButton(
```

Comment:
this will cause an increase in bundle size obviously.

Reply:
Unfortunately, yes
But I guess it's a small increase, no?